Tests/Controllers/PeopleController.cs

```diff
@@ -50,6 +51,13 @@ public Person PostPerson(string id, Person person)
         {
             person.Identifier = id;
             return person;
+        }
+
+        [HttpPost]
+        [Route("relationships/people")]
+        public JObject PostParams([FromBody] JObject bodyParams)
```

Comment:
We should probably add special case handling for deserializing into `JObject`. I'm not sure if the result should be:

1. Just pass through the request, untouched
2. Convert the request into a 'standard' web api request (bag of attributes, relationships and id).

I'm leaning towards 1, but I assume for your use-case 2 would be better?